test/gemm/gemm_fp32.cpp

```diff
@@ -32,106 +32,97 @@ namespace ck {
 namespace tensor_operation {
 namespace device {
 namespace device_gemm_instance {
-void add_device_gemm_xdl_f32_f32_f32_mk_nk_mn_instances(std::vector<DeviceGemmPtr_>&);

```

Comment:
Why removing these instancens?

Reply:
I already add the test for non shuffle version of gemm fp16, fp32 back in 856ee6fa9ca424eafff828cb7342a2716cf22273